docs/data/product/dea-intertidal/_quality.md

```diff
@@ -0,0 +1,64 @@
+| R-squared              | 0.36               | 0.74                  | **0.92**                   |
+| RMSE (m)    | 0.28               | **0.27**                  | 0.28                   |
+| MAE (m)  | 0.21               | 0.21                  | **0.20**                   |
+| Bias (m)              | 0.14               | 0.16                  | **0.12**                   |
+| Slope    | 0.44               | 0.80                  | **1.03**                   |
+
+Table 1 - DEA Intertidal Elevation validation statistics comparing performance across microtidal, mesotidal and macrotidal coastlines. 
+
+:::{figure} /_files/dea-intertidal/tiderangevalidation.*
+:alt: Validation at different tidal ranges
+
+Figure 3 – DEA Intertidal Elevation validation heatmaps comparing performance across microtidal, mesotidal and macrotidal coastlines. 
+
+## Caveats / Limitations  
+
+* Model performance improved with increasing tide range, being lowest in microtidal coastlines and highest in mesotidal and macrotidal coastlines. Due to the limited spatial extent of the intertidal zone in microtidal environments and the dominance of non-tidal water level influences (e.g. storm surge and ocean waves in high energy wave dominated environments), DEA Intertidal should be used with caution in microtidal environments. 
+
+* DEA Intertidal relies on accurate tide modelling for reliable results. Although the Ensemble Tidal Modelling approach used in this product attempts to obtain the best local tide modelling data for any given location, areas of poor quality tide modelling still remain. This is particularly the case in areas of complex and unpredictable tide dynamics, such as embayments and estuaries where global ocean tide modelling results may produce highly inaccurate outputs. In these environments, modelled elevations and exposure should be used with caution and evaluated with reference to modelled elevation uncertainty data. Examples of areas affected by poor quality tide modelling inputs include: 
+
+    * Western Port, Victoria 
+
+    *   Corner Inlet, Victoria 
+
+    * Broad Sound and Shoalwater Bay, Queensland 
+
+    * Van Diemen Gulf, Northern Territory 
+
+    * Torres Strait Islands, Queensland  
+
+* Due to biases in the tidal coverage of satellite sensors like Landsat and Sentinel-2, DEA Intertidal outputs are unlikely to cover the full extent of the intertidal zone from Lowest to Highest Astronomical tide (e.g. Figure 2). These tidal biases can be evaluated using the product’s Tidal Attribute Layers which highlight regions where DEA Intertidal will underestimate the lower, upper or full extent of the intertidal zone. 
+
+* Areas of false positive intertidal data over water exist in areas with low satellite coverage and high levels of environmental or sensor noise. Areas affected include: 
+
+    * Recherche Archipelago, Western Australia 
+
+    * South-eastern Eyre Peninsula, South Australia 
+
+    * Torres Strait Islands, Queensland 
+
```

Comment:
@stevesagar We should add something like:
```
* Areas of highly turbid water can be incorrectly mapped as intertidal terrain, particularly if temporal patterns of turbidity correlate with tidal dynamics. Areas affected by turbidity include:
    * King Sound, Western Australia
    * Cambridge Gulf, Western Australia
    * Joseph Bonaparte Gulf, Northern Territory
    * Daly River, Northern Territory
    * Broad Sound, Queensland
```

Reply:
agreed, done